### ISSUE TITLE
Oh boy, another tf2 attributes issue

### ISSUE THREAD

**Issue by Sluethen** [FlaminSarge/tf2attributes#26]
I've looked around in the issues, didnt find anything similar to my issue. Might have missed something though.

```L 06/24/2018 - 20:30:07: SourceMod error session started
L 06/24/2018 - 20:30:07: Info (map "pl_upward") (file "errors_20180624.log")
L 06/24/2018 - 20:30:07: [tf2attributes.smx] Could not initialize call to CEconItemSchema::GetItemDefinition
L 06/24/2018 - 20:30:07: [tf2attributes.smx] Could not initialize call to CEconItemView::GetSOCData
L 06/24/2018 - 20:30:07: [tf2attributes.smx] Could not initialize call to CEconItem::SetCustomName
L 06/24/2018 - 20:30:07: [SM] Exception reported: Could not initialize call to GEconItemSchema
L 06/24/2018 - 20:30:07: [SM] Blaming: tf2attributes.smx
L 06/24/2018 - 20:30:07: [SM] Call stack trace:
L 06/24/2018 - 20:30:07: [SM]   [0] SetFailState
L 06/24/2018 - 20:30:07: [SM]   [1] Line 131, /home/forums/content/files/8/4/3/0/4/116848.attach::OnPluginStart
L 06/24/2018 - 20:30:07: [SM] Unable to load plugin "tf2attributes.smx": Error detected in plugin startup (see error logs)
L 06/24/2018 - 20:30:07: [SM] Unable to load plugin "tf2attributes_ubup.smx": Could not find required plugin "tf2attributes"
L 06/24/2018 - 20:30:16: [SM] Exception reported: Script execution timed out
L 06/24/2018 - 20:30:16: [SM] Blaming: tf2itemsinfo.smx
L 06/24/2018 - 20:30:16: [SM] Call stack trace:
L 06/24/2018 - 20:30:16: [SM]   [1] Line 1544, C:\Users\Brandon\Desktop\sourcemod\scripting\tf2itemsinfo.sp::PrecacheItemSchema
L 06/24/2018 - 20:30:16: [SM]   [2] Line 253, C:\Users\Brandon\Desktop\sourcemod\scripting\tf2itemsinfo.sp::OnPluginStart
L 06/24/2018 - 20:30:16: [SM] Unable to load plugin "tf2itemsinfo.smx": Error detected in plugin startup (see error logs)
L 06/24/2018 - 20:38:33: Error log file session closed.
L 06/24/2018 - 20:38:58: SourceMod error session started
L 06/24/2018 - 20:38:58: Info (map "pl_upward") (file "errors_20180624.log")
L 06/24/2018 - 20:38:58: [SM] Exception reported: Script execution timed out
L 06/24/2018 - 20:38:58: [SM] Blaming: tf2itemsinfo.smx
L 06/24/2018 - 20:38:58: [SM] Call stack trace:
L 06/24/2018 - 20:38:58: [SM]   [1] Line 1654, C:\Users\Brandon\Desktop\sourcemod\scripting\tf2itemsinfo.sp::GetAttribIDByName
L 06/24/2018 - 20:38:58: [SM]   [2] Line 1527, C:\Users\Brandon\Desktop\sourcemod\scripting\tf2itemsinfo.sp::PrecacheItemSchema
L 06/24/2018 - 20:38:58: [SM]   [3] Line 253, C:\Users\Brandon\Desktop\sourcemod\scripting\tf2itemsinfo.sp::OnPluginStart
L 06/24/2018 - 20:38:58: [SM] Unable to load plugin "tf2itemsinfo.smx": Error detected in plugin startup (see error logs)
L 06/24/2018 - 20:39:13: Error log file session closed.
```

I'm running windows with a fresh install of the server, sourcemod, meta mod, and all of the stuff.

If you need more info let me know.

**Comment by Benoist3012** [comment]
This means that the signatures are broken. Are you using this gamedata file https://github.com/FlaminSarge/tf2attributes/blob/master/tf2.attributes.txt and not the one on the alliedmodders forum? The gamedata on alliedmodders is extremely outdated and shouldn't be used.

**Comment by Sluethen** [comment]
Hrm. I didn't even find one on the allied modders fourm, but i'll reinstall that,

Would it hurt if I just copy & pasted the contents inside?

**Comment by Benoist3012** [comment]
Nah you can just copy paste

**Comment by Sluethen** [comment]
hrm

Still not working.

Maybe its because i compiled the .sp file using the sourcemod page?

letme try getting the github release.(That i didnt see earlier)

**Comment by Benoist3012** [comment]
Yup the one on allied is extremely outdated

**Comment by FlaminSarge** [comment]
Latest version on GitHub is 1.3.2 but latest on AM is 1.2.1 because I haven't had the time to do a formal release yet...

**Comment by Sluethen** [comment]
Ah.
rip

Anyways, Any possible solution to this?

**Comment by FlaminSarge** [comment]
The latest GitHub version _should_ be fine on Linux. Not sure about Windows.

**Comment by Sluethen** [comment]
Ah. Im on windows so that might be it, so just wait for an update?

**Comment by Sluethen** [comment]
Ah, Should I close the issue?

**Comment by Sluethen** [comment]
Update:

New error

now the script timesout

**Comment by Sluethen** [comment]
Sometimes anyways

**Comment by Sluethen** [comment]
Hrm. My main issue seemed to have fixed when updating a signature in issue #27 

The only thing left is the timeout. Maybe the tf2.attributes is outdated?
